etl/data_enhancement/domain/default.py

```diff
@@ -75,6 +76,5 @@ def rank_tags(file):
 # Enhance the geo_location
 def add_lat_lon(file):
     enhancer = LatLonEnhancer()
-
     for post in file:
-        post = enhancer.enhance(post)
+        enhancer.enhance(post)
```

Comment:
Please remove the return value from enhance. It is not needed anymore.

Reply:
Done. You have already changed it.

Reply:
It is not changed. lat_lon_enhancer.py returns post in the enhance function (line 34) which is never used.

Reply:
Sorry. I thought I should change something in the displayed line.